app/views/statistics/index.html.erb

```diff
@@ -0,0 +1,95 @@
+<div class="small-12 column">
+  <h2><%=t("statistics.statistics")%></h2>
+  <p><%=t("statistics.description")%></p>
+
+  <button class="accordion" style="width: 100%;"><%=t("statistics.category_lobbies")%></button>
+  <div class="panel" style="display: none;">
+  <table class="display" cellpadding="0" width="100%" >
+    <thead>
+      <tr>
+        <th width="40%"><%= t('statistics.category')%></th>
+        <th width="15%"><%= t('statistics.amount')%></th>
+      </tr>
+    </thead>
+    <tbody>
+      <%@lobby_category.each do|name , number| %>
+      <tr>
```

Comment:
Review indentation and spacing!

Reply:
Make proper usage of Zurb Foundation accordion components. It should be initialized by default!

Reply:
I changed the interface by the indicated one